modules.d/01bootetc/boot-etc.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+. /lib/dracut-lib.sh
+
+bootetc=$(getarg bootetc=)
+if [ "${bootetc}x" == "x" ]; then
+    exit 0
+fi
+
+mount_boot bootetc
+if [ -d /boot/initrd-etc ] && [ ! -f /run/bootetc.done ]; then
+    info "bootetc: Updating initramfs etc from ${bootetc}/initrd-etc"
+    copytree /boot/initrd-etc /etc
+    touch /run/bootetc.done
+
+    if [ -z "$DRACUT_SYSTEMD" ]; then
+        systemctl try-restart dracut-cmdline.service
+        systemctl try-restart systemd-udev-trigger.service
```

Comment:
Hm, why does udev need to be retriggered?

Reply:
probably so the device that contains the boot partition can be found?

Reply:
oh `re-triggered`.. not sure

Reply:
Udev rules :)